src/foam/nanos/notification/email/ApplyEmailTemplateService.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.notification.email',
+  name: 'ApplyEmailTemplateService',
```

Comment:
I'd call this EmailTemplateApplyEmailPropertyService since it's an EmailPropertyService that does an EmailTemplate.apply